modules/customizing-the-web-console-URL.adoc

```diff
@@ -3,67 +3,60 @@
 // * web_console/customizing-the-web-console.adoc
 
 [id="customizing-the-web-console-url_{context}"]
-= Customizing the web console URL
+= Customizing console routes
 
-You can update the web console URL, `consoleURL`, to a custom value.
+For `console` and `downloads` routes, custom routes functionality is now implemented to use the new `ingress` config route configuration API. The Console Operator already contained custom route customization, but for the `console` route only. Now, if the `console` custom route is set up in both the `ingress` config and `console-operator` config, then the new `ingress` config custom route configuration takes precedent, because the route configuration via `console-operator` config is being deprecated.
```

Comment:
The use of "now," "new," and "already" are a fit for the release notes, but it can read better if you switch all to present tense,,,"...custom routes functionality uses the `ingress` config route configuration API."

"via" receives a caution icon in ISG p428 and can prolly be replaced with "...route configuration with the `console-operator` config is..." Hmm, I guess it's either deprecated now or maybe "will be deprecated in a future release."

Third sentence, "then" in if/then is also a caution, courtesy ISG p342.  My least favorite rule, but obliged to point it out.

I also think you can split the third sentence into two sentences at ", because."